### PR TITLE
40 add parser junit category ignore

### LINKED ISSUE #40: Add Junit Testing for AddParser, allow for optional Category
related to #37

### PR DESCRIPTION
**Pull request by yeozhishen**
Fixes #40

### CODE REVIEW

File: src/test/java/brokeculator/parser/AddParserTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        assertInstanceOf(AddCommand.class, resultCommand);
+    }
+    @Test
+    void parseInput_stringNoCategory_addCommand() {
+        String input = "add /n test1 /d 1900 /a 10";
+        Command resultCommand = AddParser.parseInput(input);
+        assertInstanceOf(AddCommand.class, resultCommand);
+    }
+    @Test
+    void parseInput_stringInvalidFormat_invalidCommand() {
```
Comment (JianJiaT) [inline src/test/java/brokeculator/parser/AddParserTest.java:24]: Wouldn't we expect a AddCommand with "null" category? Also, is this not a valid format where the user inputs /c but without a category?

**Review by JianJiaT — COMMENTED** [review]
Looks good aside from a slight concern with one of the tests

File: src/test/java/brokeculator/parser/AddParserTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        String input = "add /n test1 /d 1900 /a 10 /c";
+        Command resultCommand = AddParser.parseInput(input);
+        assertInstanceOf(AddCommand.class, resultCommand);
+    }
```
Comment (JianJiaT) [inline src/test/java/brokeculator/parser/AddParserTest.java]: Wouldn't we expect a AddCommand with "null" category? Also, is this not a valid format where the user inputs /c but without a category?

**Review by JianJiaT — COMMENTED** [review]
Looks good aside from a slight concern with one of the tests

File: src/test/java/brokeculator/parser/AddParserTest.java (hunk 2 above)
Comment (yeozhishen) [reply src/test/java/brokeculator/parser/AddParserTest.java]: apologies the function name was not edited correctly

**Pull request merged**